Gui/GUIutils/settings.py

```diff
@@ -159,38 +160,17 @@ def __init__(self, DBhostname, DBIP, DBName):
     }
 
 BoxSize = {
-    "SingleSCC": 1,
+    "TFPX SCC": 1,
     "TFPX Quad": 4,
     "TEPX Quad": 4,
     "TBPX Quad": 4,
-    "CROC SCC": 1,
-    "CROC 1x2": 2,
+    "TFPX CROC SCC": 1,
+    "TFPX CROC 1x2": 2,
     "TFPX CROC Quad": 3,
 }
 
-
-ConfigFiles = {
-    "Latency": "/Configuration/Defaults/CMSIT.xml",
-    "PixelAlive": "/Configuration/Defaults/CMSIT.xml",
-    "NoiseScan": "/Configuration/Defaults/CMSIT.xml",
-    "SCurveScan": "/Configuration/Defaults/CMSIT.xml",
-    "GainScan": "/Configuration/Defaults/CMSIT.xml",
-    "ThresholdEqualization": "/Configuration/Defaults/CMSIT.xml",
-    "GainOptimization": "/Configuration/Defaults/CMSIT.xml",
-    "ThresholdMinimization": "/Configuration/Defaults/CMSIT.xml",
-    "ThresholdAdjustment": "/Configuration/Defaults/CMSIT.xml",
-    "InjectionDelay": "/Configuration/Defaults/CMSIT.xml",
-    "ClockDelay": "/Configuration/Defaults/CMSIT.xml",
-    "BitErrorRate": "/Configuration/Defaults/CMSIT.xml",
-    "DataRBOptimization": "/Configuration/Defaults/CMSIT.xml",
-    "ChipIntVoltageTuning": "/Configuration/Defaults/CMSIT.xml",
-    "GenericDAC-DAC": "/Configuration/Defaults/CMSIT.xml",
-    "Physics": "/Configuration/Defaults/CMSIT.xml",
-    "AllScan": "/Configuration/Defaults/CMSIT.xml",
-}
-
-Test = {
-    "AllScan_Tuning": "noise",
+'''Test = {
```

Comment:
We should not be block comment things out. This leads to confusion down the line when trying to determine whether or not something was a necessary piece of code.